### PR TITLE
Regenerate with latest gir

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: src/auto/key_file.rs [hunk 1]
```diff
@@ -223,7 +223,7 @@ impl KeyFile {
         }
     }
 
-    //pub fn set_boolean_list(&self, group_name: &str, key: &str, list: /*Unimplemented*/&CArray TypeId { ns_id: 0, id: 1 }, length: usize) {
+    //pub fn set_boolean_list(&self, group_name: &str, key: &str, list: /*Unimplemented*/&CArray TypeId { ns_id: 0, id: 1 }) {
```
Comment (sdroege) [inline src/auto/key_file.rs]: set_string_list() will be fixed with the next GLib .gir update. set_*_list() and get_*_list() requires non-ptr container support.

**Comment by EPashkin** [comment]
Seems `g_filename_from_uri` need be manual as `filename_to_uri`

**Comment by sdroege** [comment]
Done. But I'm wondering, the UTF8 variants use UTF8 for filenames/paths but std::path::PathBuf is in the local encoding, or not?

Also this function could as well be hidden, the Rust url crate has this as a pure Rust implementation.

**Comment by EPashkin** [comment]
Checked that function works on Windows.
https://github.com/gtk-rs/glib/blob/master/src/translate.rs#L399-L410 
and  https://github.com/gtk-rs/glib/blob/master/src/translate.rs#L782-L793 handles conversion.
IMHO better don't ignore, but final decision on @GuillaumeGomez 
Thanks, @sdroege

**Comment by GuillaumeGomez** [comment]
Tomorrow I'm on a plane so if you don't have answer from me, please ping me.

**Comment by sdroege** [comment]
Ok with me to keep it, it just doesn't seem very useful to me :)

**Comment by sdroege** [comment]
Related to https://github.com/gtk-rs/gir/issues/376

**Comment by GuillaumeGomez** [comment]
Well, keeping them doesn't hurt so unless there's a huge reason to remove them, I'm more in favor to keep them.

**Comment by EPashkin** [comment]
@sdroege, as It still don't merged, can you apply https://github.com/gtk-rs/gir/pull/392 too ?

**Comment by sdroege** [comment]
Sure, and also the others

**Comment by sdroege** [comment]
Done

**Comment by EPashkin** [comment]
Thanks again, @sdroege 
👍 for merge

**Comment by GuillaumeGomez** [comment]
Thanks!

**Comment by sdroege** [comment]
Oops, sorry! https://github.com/gtk-rs/glib/pull/185

The other pull requests got updated to also have gir updated correctly.

**Pull request merged**